src/polymer-project.ts

```diff
@@ -327,6 +327,24 @@ class HtmlSplitter extends Transform {
           this.push(scriptFile);
         }
 
+        for (let i = 0; i < styleTags.length; i++) {
+            let styleTag = styleTags[i];
+            let source = dom5.getTextContent(styleTag);
+            let extension = 'css';
+            let childFilename = `${osPath.basename(filePath)}_style_${i}.${extension}`;
+            let childPath = osPath.join(osPath.dirname(filePath), childFilename);
+            styleTag.childNodes = [];
+            dom5.setAttribute(styleTag, 'src', childFilename);
```

Comment:
True `<link include=>` isn't valid, and so we shouldn't generate that :)

If a `style` tag has a `include` attribute we have to be more careful about what we generate. Possibly a `<style include=><style>` _and_ a `<link rel=stylesheet>`.

@sorvell is there any valid transformation that externalizes a `<script include=>` tag?